frontend/src/features/rollout-announcement/RolloutAnnouncementModal.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import { BiRightArrowAlt } from 'react-icons/bi'
+import {
+  Box,
+  Modal,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react'
+
+import Button from '~components/Button'
+
+import { LastFeatureContent } from './components/LastFeatureContent'
+import { NewFeatureContent } from './components/NewFeatureContent'
+import { ProgressIndicator } from './components/ProgressIndicator'
+import { NEW_FEATURES, OTHER_UPDATES } from './Announcements'
+
+interface RolloutAnnouncementModalProps {
+  isOpen: boolean
+  onClose: () => void
+}
+
+export const RolloutAnnouncementModal = ({
+  isOpen,
+  onClose,
+}: RolloutAnnouncementModalProps): JSX.Element => {
+  const [currActiveIdx, setCurrActiveIdx] = useState<number>(0)
+
+  const numOtherUpdatesPages = 1
+  const NUM_NEW_FEATURES = NEW_FEATURES.length + numOtherUpdatesPages
```

Comment:
let's be consistent about casing for constants, seems like `numOtherUpdatesPages` should also be `UPPER_SNAKE_CASE`. also we can define these outside the component.

Reply:
Updated the PR to fix this